rmw_fastrtps_cpp/src/rmw_subscription.cpp

```diff
@@ -148,6 +148,32 @@ rmw_subscription_get_actual_qos(
   return rmw_fastrtps_shared_cpp::__rmw_subscription_get_actual_qos(subscription, qos);
 }
 
+rmw_ret_t
+rmw_subscription_set_content_filter(
+  rmw_subscription_t * subscription,
+  const rmw_subscription_content_filter_options_t * options)
+{
+  // Unused in current implementation.
+  (void) subscription;
+  (void) options;
+  RMW_SET_ERROR_MSG("unimplemented");
```

Comment:
what will happen in rclcpp when the rmw implementation doesn't support content filtering?

Reply:
`NOT SUPPORT` will be returned to application for now. @iuhilnehc-ynos if i am mistaken, let us know your thought.

Reply:
As `rclcpp` always did, it'll throw an exception (RCLError, I will add a new exception named RCLUnsupport) if users call `set_content_filter` without checking `is_cft_enabled`.

https://github.com/ros2/rclcpp/blob/f509a3469b790fd80311a89e2931b420e7ca9a16/rclcpp/src/rclcpp/subscription_base.cpp#L369-L375